src/Altinn.App.PlatformServices/Implementation/AppOptionsService.cs

```diff
@@ -1,6 +1,7 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using Altinn.App.Common.Models;
+using Altinn.App.PlatformServices.Models;
 using Altinn.App.PlatformServices.Options;
 
 namespace Altinn.App.PlatformServices.Implementation
```

Comment:
many of the services in platformServices have been moved to the namespace `Altinn.App.Services.Implementation`
Would that be the correct place for this service as well? Not sure what the final names will be, but I guess we should try to be consistent here either way

Reply:
I'll have a look.